integration-tests/keycloak-authorization-wiremock/pom.xml

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-integration-test-keycloak-authorization-wiremock</artifactId>
+    <name>Quarkus - Integration Tests - Keycloak Policy Enforcer Wiremock</name>
+    <description>Module that contains Keycloak Policy Enforcer related tests</description>
+
+    <properties>
+        <keycloak.url>http://localhost:8180/auth</keycloak.url>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-keycloak-authorization</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-mutiny</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-keycloak-admin-client</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jsonb</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-oidc</artifactId>
+        </dependency>
+
+        <!-- test dependencies -->
+        <dependency>
```

Comment:
Hi @cemnura as discussed earlier - please drop this one and for now come up with a simpler local Wiremock code - drop everything related to the opaque/introspection and code flow tokens - it will be easier for you to handle - you'd start incrementally adding stubs one by one to fix `keycloak-authorization` tests (I'd disable all by a single test and move on from there)
thanks